### ISSUE TITLE
Always return all fields for a project

### ISSUE THREAD

**Issue by mkosmala** [zooniverse/Panoptes#702]
In retrieving a Project, it's "science case" text field is blank. For example, as part of the project 'Drawing', I get these fields:
```
'title': 'Drawing', 
'introduction': '', 
'created_at': '2015-03-06T19:11:47.984Z', 
'faq': '', 
'updated_at': '2015-03-06T19:11:47.984Z',
'classifications_count': 1, 
'guide': '', 
'private': True, 
'retired_subjects_count': 0, 
'avatar': None, 
'available_languages': ['en-us'], 
'education_content': '', 
'science_case': '', 
'id': '62', 
'description': 'draw me'
```

science_case is listed as '', but I can clearly see the science case on Panoptes-FrontEnd.

**Comment by edpaget** [comment]
No point to try to save query time since the content is eager loaded anyway now.

**Comment by edpaget** [comment]
This is the same as #718

**Comment by edpaget** [comment]
Closed via #724
